### PR TITLE
AEROGEAR-3587 updated readme

### PR DESCRIPTION
**Pull request by StephenCoady**
Motivation: https://issues.jboss.org/browse/AEROGEAR-3587

### CODE REVIEW

**Comment by StephenCoady** [comment]
@ziccardi mind taking a look to see if it aligns with what you are doing on the Xamarin one?

File: readme.adoc [hunk 1]
```diff
@@ -23,7 +35,34 @@ A Native iOS Application that demonstrates features of the Aerogear iOS SDK.
...
 - Push notifications with UPS server
 
-== Work with Self-signed Certificate
+== Build Instructions
+
+=== Installing Dependencies
+To install all pods, run the following command:
+
+```
+pod install
```
Comment (wei-lee) [inline readme.adoc:44]: can we add a pre-req section and list the SDKs/IDEs the user need to install before building the app.

File: readme.adoc [hunk 2]
```diff
@@ -23,7 +35,34 @@ A Native iOS Application that demonstrates features of the Aerogear iOS SDK.
...
+
+
+=== Troubleshooting
+
+==== Enabling Push Capabilites
+
+If you wish to use push on the showcase application you will need to enable the push capability. 
+To do this, go to ios-showcase-template and under `Capabilities` enable `Remote Notifications`
+
+NOTE: You will need an Apple developer account with the push capability for this.
```
Comment (wei-lee) [inline readme.adoc:58]: we should also mention this also doesn't work on the simulator

File: readme.adoc [hunk 3]
```diff
@@ -23,7 +35,34 @@ A Native iOS Application that demonstrates features of the Aerogear iOS SDK.
...
+
+==== Running on a real device
+If you are running the application on a real device you will need to create a provisioning 
+profile. To do this, go to ios-showcase-template and under `General` and then `Signing` select your desired 
+profile if you already have one, and if not click `Add an Account` to create one.
+
+==== Work with Self-signed Certificate
```
Comment (wei-lee) [inline readme.adoc:67]: link to https://docs.aerogear.org/aerogear/latest/getting-started.html#using-self-signed-certificates-in-mobile-apps

**Review by wei-lee — CHANGES_REQUESTED** [review]
see my comments

File: readme.adoc (hunk 1 above)
Comment (StephenCoady) [reply readme.adoc:44]: done

File: readme.adoc (hunk 2 above)
Comment (StephenCoady) [reply readme.adoc:58]: done

File: readme.adoc (hunk 3 above)
Comment (StephenCoady) [reply readme.adoc:67]: done

**Comment by StephenCoady** [comment]
@wei-lee mind taking a look again, addressed your comments.

**Review by wei-lee — APPROVED** [review]

**Pull request merged**